src/java.base/macosx/classes/sun/nio/fs/MacOSXFileSystem.java

```diff
@@ -35,6 +36,17 @@
 
 class MacOSXFileSystem extends BsdFileSystem {
 
+    private static final String PROPERTY_ENABLE_FILE_NAME_ENCODING =
+        "jdk.nio.path.useNormalizationFormD";
```

Comment:
Do we document system properties like this one in some javadoc through the use of `@systemProperty` so that it shows up in the documentation? Or is it OK to just have this mentioned in the release notes?